views/dashboard/about-page/components/FormView.tsx

```diff
@@ -0,0 +1,201 @@
+import { use, useEffect } from 'react'
+import { useDispatch } from 'react-redux'
+import useForm from '@/utils/hooks/useForm'
+import FormValidations from '@/utils/validations/forms'
+import { FileUpload, FormButtonGroup, TextField } from '@/components/form'
+import { updateAboutPageData } from '@/lib/db/cms/about-page'
+import { setAbout } from '@/lib/redux/slices/aboutPage'
+import { StyledForm } from './StyledFormView'
+import { handleCloudUpload } from '@/utils/fileConversions'
+import { enqueueSnackbar } from 'notistack'
+import { statuses, messages } from '@/utils/banners'
+
+const initialState = {
+    name: { value: '', error: '' },
+    description: { value: '', error: '' },
+    header: { value: '', error: '' },
+    subHeader: {
+        value: '',
+        error: ''
+    },
+    img: [
```

Comment:
Initial state for img should still be `img: { value: [] }`. We always want to tie it to the right object structure, and then initially it is an empty array as the value

Reply:
Ahh you right lets fight for 10mins tho